lib/modules/accountSwitcher.js

```diff
@@ -142,13 +149,33 @@ modules['accountSwitcher'] = {
 
 							RESUtils.getUserInfo(function(userInfo) {
```

Comment:
This section ought to be refactored instead of tacking on copy-pasta.  Instead of calling RESUtils.getUserInfo here, we ought to call modules['accountSwitcher'].updateUserDetails() after the for loop.  Can you take care of that?